docs/collecting-data/code-generation/commands/index.md

```diff
@@ -15,7 +15,7 @@ This page only summarizes the CLI commands and the options for each command. For
 
 ### `snowtype init`
 
-Initialize the setup of Snowtype code generation in a project. Creates the `snowtype.config.json` file.
```

Comment:
Maybe we can change the `init` and `generate` statements to just `configuration file` versus adding all the options .

E.g:
`Initialize the setup of Snowtype code generation in a project. Creates the Snowtype configuration file.`

`Generates tracking code based on the Snowtype configuration file. Can generate/modify the `.snowtype-lock.json` file.`

WDYT ?